HIP/hip-260.md

```diff
@@ -122,6 +126,63 @@ final updated value of the storage slot after the completion of the smart
 contract call. Transient states between the start and finish of the contract are
 not stored in the record stream.
 
+### Record Stream File Changes
+
+Currently Hedera nodes produce record stream files that contain a series of
+transactions within a two second interval. The record stream file (.rcd) 
+contains a transaction record for each transaction in that file. For each 
+record stream file there is a corresponding signature file (rcd_sig) 
+that includes the signature generated by the node.
+
+#### Side car records file
+
+A new file of type (.rcd_side) will be introduced to track all side car records
+produced during the same 2 second period as the main record file (.rcd). All
+side car records in the .rcd_side file should have a corresponding main record
+with the same id in the main record file (.rcd).
+
+The sidecar + record file + signature file will all be zipped together into a
+single zip and uploaded to the AWS/Google buckets.
+
+The side car file will have the following structure:
+
+| Name                              | Type (Bytes) |  Description                                                     |
+| --------------------------------- | -----------: | ---------------------------------------------------------------: |
+| Record Stream File Format Version |    int (4)   | "Record stream file format version, Value: 5"                    |
+| Hapi Proto Major Version          |    int (4)   | Matches NetworkGetVersionInfo.hapiProtoVersion Major.Minor.Patch |
```

Comment:
I think the Description for the first row has overflowed to the second row and has shifted all other descriptions with one row and now there is a misalignment.

Reply:
This should not be a problem when looking at the rendered markdown